root/edit/components/EditArtwork.js

```diff
@@ -33,7 +33,7 @@ const EditArtwork = ({
 
   return (
     <tr>
-      <th>{l('Cover art:')}</th>
+      <th>{addColonText(lp('Cover art', 'front cover display'))}</th>
```

Comment:
This component is used for displayed cover art edits, not just front covers. It doesn’t seem to be any different from other header usages.

```suggestion
      <th>{addColonText(lp('Cover art', 'header'))}</th>
```

Reply:
Actually, the main relevant difference for "Cover art" seems to be if it means one image or many, so I'm changing these to "singular", the others (mostly "header" ones) to "plural".